modules/backend/formwidgets/Relation.php

```diff
@@ -123,6 +123,18 @@ protected function makeRenderFormField()
             // by joining its pivot table. Remove all joins from the query.
             $query->getQuery()->getQuery()->joins = [];
 
+            // Remove sorting on columns of joined tables including pivots
+            $pivotColumns = $this->model->$relationType[$relationObject->getRelationName()]['pivot'] ?? [];
```

Comment:
This line will throw an exception for belongsToOne relationship because getRelationName() doesn't exist for those.